includes/WikiPage.php

```diff
@@ -1602,6 +1600,8 @@ public function doEdit( $text, $summary, $flags = 0, $baseRevId = false, $user =
 		wfRunHooks( 'ArticleSaveComplete', array( &$this, &$user, $text, $summary,
 			$flags & EDIT_MINOR, null, null, &$flags, $revision, &$status, $baseRevId ) );
 
+		$user->incEditCount();
```

Comment:
This will now cause the edit count to be updated on null edits when it should not be. The previous version only increments the edit count on editing an existing article if the content is changed.